docs/configuration/tooltip.md

```diff
@@ -161,10 +161,16 @@ The tooltip items passed to the tooltip callbacks implement the following interf
 
 ```javascript
 {
-    // X Value of the tooltip as a string
+    // Label for the tooltip
+    label: String,
+
+    // Value for the tooltip
+    value: String,
+
+    // (deprecated) X Value of the tooltip
```

Comment:
Maybe you could also say "Use 'label' or 'value' instead". Same below

Reply:
Done